extensions/markdown-navigator/package.json

```
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "markdown-navigator",
+  "title": "Markdown Navigator",
+  "description": "Efficiently manage and navigate your Markdown files.",
+  "icon": "extension-icon.png",
+  "author": "chihkang",
+  "categories": [
+    "Productivity",
+    "Documentation"    
+  ],
+  "license": "MIT",
+  "commands": [  
+    {
+      "name": "markdown-navigator",
+      "title": "Markdown Navigator",
+      "description": "Browse, search and manage your Markdown files",
+      "mode": "view",
+      "subtitle": "Markdown",
+      "icon": "extension-icon.png"
+    }    
+  ],
+  "permissions": [
+    "file-system"
+  ],
```

Raycast does not support such fields. Let's remove it.

```suggestion
  ],
```

---

@LitoMore 👍